pyproject.toml

```diff
@@ -55,3 +55,10 @@ django = ["django"]
 
 [tool.ruff.lint.mccabe]
 max-complexity = 20
+
+[tool.uv]
+dev-dependencies = [
+  "django-extensions==3.2.3",
```

Comment:
I don't really like using `django-extensions` so -1

simply installing `ipython` will drop you to ipython shell anyway. we can keep that